Fix "by using" in concepts N--Z (#3207)
theforeman/foreman-documentation: guides/common/modules/con_retrieving-metrics.adoc

[id='retrieving-metrics_{context}']
= Retrieving metrics

You can retrieve metrics from PCP by using the CLI or the web UI.

> A number of CLI tools are provided with PCP, which can either output live data, or data from archived sources.

Author: ```suggestion
Several CLI tools are provided with PCP, which can either output live data, or data from archived sources.
```
